test/setup.ts

```diff
@@ -24,4 +24,5 @@ export default () => {
   //  feature flags
   process.env.SHOW_SERVICE_OFFLINE_PAGE = "false";
   process.env.FEATURE_FLAG_ENABLE_SAVE_AND_RESUME_17102022 = "true";
+  process.env.FEATURE_FLAG_ENABLE_ROE_UPDATE_24112022 = "true";
```

Comment:
This will enable update feature for all tests?

Reply:
Might be something we do separately from this PR.

Reply:
@AndreKramer7654 , one of the tests was showing the service offline page as the FT was not enabled, can remove it if needed...

Reply:
Can mock the feature flag in the test?